packages/backend/src/webhook/service.test.ts

```diff
@@ -93,6 +96,94 @@ describe('Webhook Service', (): void => {
     })
   })
 
+  describe('getPage', (): void => {
+    getPageTests({
+      createModel: () => createWebhookEvent(deps),
+      getPage: (pagination?: Pagination) =>
+        webhookService.getPage({ pagination })
+      // TODO: Make this work with filtering? eg: webhookService.getPage({ pagination, type: 'event1' })
+      // Currently does not work with filters unless you increase the # of models created
+      // in getPageTests such that there are ~20+ samples for the event type when filtering.
+      // Not a great fix, because technically increasing amount of models created doesnt garuntee
+      // enough isntances of each event type. Would probably be best to somehow pass in created models
+      // or override function to create models in beforeEach.
```

Comment:
I think just make sure to test the filters sufficiently in this file (like you have below), and we won't need to update `getPageTests` IMO

Reply:
Still thinking about your comment here... 

Originally my concern was "is there some combination of pagination args and filter args where the page/cursor will be incorrect?" So my thinking was adding filtering to the `getPageTests` would be an exhaustive way to test that. It is certainly important that filters are applied before pagination, but beyond that I dont see a way filtering could impact pagination. Which points towards just keeping the filter test below. I'm just trying to validate I am using a good pattern for that. I suppose an additional filters can just be added to the same `Can paginate and filter` test.

Probably moot, but for whatever it's worth, I think there is a better way to accomplish what I describe in my comment here. Instead of changing the `getPageTests` I could make the `createModel` function we pass in here make a new model for each test webhook event type.